src/main/java/pages/IssuePage.java

```diff
@@ -0,0 +1,63 @@
+package pages;
+
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Selectors.byId;
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+import static com.codeborne.selenide.Selenide.open;
+
+public class IssuePage {
+
+    String commentButtonID = "footer-comment-button";
+    String textTabButtonID = "aui-uid-1";
+    String commentTextAreaXPath = "//*[@id=\"comment-wiki-edit\"]/textarea";
+    String addCommentSubmitButtonID = "issue-comment-add-submit";
+    String issueActionsContainerID = "issue_actions_container";
+    String issueActionsTextXPath = "//*[@id=\"issue_actions_container\"]/div/div/div[@class=\"action-body flooded\"]";
+    String priorityElementID = "priority-val";
+    String priorityFieldID = "priority-field";
+    String priorityFormSubmitButtonXPath = "//form[@id='priority-form']/div[@class='save-options']/button[@type='submit']";
+
+    public void navigateTo(String url){
+        open(url);
+    }

```

Comment:
Rewrite the format selectors from:
public void navigateTo(String url){
open(url);
}

to:
public void navigateTo(String url){open(url); }

Reply:
Suggestion from Artyom might help to make the code more beautiful.

Reply:
But maybe it's better to leave the current version to save general code style